distribution.pm

```diff
@@ -265,7 +265,13 @@ Define the patterns to look for in the serial console.
 The patterns can be either I<hard> or I<soft>.
 
 Example:
-    set_expected_serial_failures(soft=>[qr/Pattern1/], hard=>[qr/Pattern2/]);
+    set_expected_serial_failures(soft => [
```

Comment:
I guess it would be beneficial to have test for this, as it will show how to use it. Not sure if we can assert soft-failures though. See https://github.com/os-autoinst/os-autoinst/blob/master/t/99-full-stack.t